checkpoint/data_source_checkpoint_management_simple_cluster.go

```diff
@@ -37,6 +37,12 @@ func dataSourceManagementSimpleCluster() *schema.Resource {
 				Computed:    true,
 				Description: "Cluster mode.",
 			},
+			"geo_mode": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: "Enable Geo mode for cross-az clusters.",
```

Comment:
change description to:
"Cluster High Availability Geo mode.
This setting applies only to a cluster deployed in a cloud. Available when the cluster mode equals "cluster-xl-ha"."

Reply:
Implemented